tests/Benchmarks/System.Text.Primitives/PrimitiveParserBoolPerfTests.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers.Text;
+using System.Collections.Generic;
+using BenchmarkDotNet.Attributes;
+using Benchmarks.System.Text.Primitives;
+
+namespace System.Text.Primitives.Benchmarks
+{
+    public partial class PrimitiveParserPerfTests
+    {
+        [Benchmark]
```

Comment:
Can we mark this benchmark as baseline?

Reply:
there can be only 1 baseline benchmark per type so if you split `PrimitiveParserPerfTests` into multiple types then yes